git-tar/function/ops.go

```diff
@@ -316,91 +317,98 @@ func deploy(tars []tarEntry, pushEvent sdk.PushEvent, stack *stack.Services, sta
 	c := http.Client{}
 	gatewayURL := os.Getenv("gateway_url")
 
+	sem := make(chan int, 4)
```

Comment:
sem?

Reply:
This is for semaphore. Does it look clear or I should better rename?

Reply:
It's not a semaphore it's a channel. Consider a more meaningful name. workChannel? (I don't know)